src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -442,7 +442,7 @@ static bool _isValidImageMimeTypeAndEncoding(const char** href, const char** mim
 }
 
 
-static unique_ptr<Picture> _imageBuildHelper(SvgNode* node, float vx, float vy, float vw, float vh)
+static unique_ptr<Picture> _imageBuildHelper(SvgNode* node, float vx, float vy, float vw, float vh, string svgPath)
```

Comment:
we can avoid memory copy in the parameter,
probably =>
const string& svgPath;
or
const char* svgPath;

Reply:
of course, changed